doc/source/release-notes/Percona-Server-8.0.12-2rc1.rst

```diff
@@ -0,0 +1,93 @@
+.. rn:: 8.0.12-2rc1
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Following the `alpha release announced earlier
+<https://www.percona.com/blog/2018/09/27/announcement-alpha-build-of-percona-server-8-0/>`_,
+Percona announces the release candidate of Percona Server for MySQL |release| on
+October 31, 2018. Download the latest version from the Percona web site or the
+Percona Software Repositories.
+
+This release is based on |MySQL| 8.0.12 and includes all bug fixes in it. It is
+a *Release Candidate* quality release and it is not intended for
+production. If you want a high quality, Generally Available release, use the
+current Stable version (the most recent stable release at the time of writing in
+the 5.7 series is 5.7.23-23).
+
+Percona provides completely open-source and free software.
+
+Installation
+================================================================================
+
+As this is a release candidate, installation is performed by enabling the
+testing repository and installing the software via your package manager.  For
+Debian based distributions, see `apt installation instructions
+<https://www.percona.com/doc/percona-server/8.0/installation/apt_repo.html>`_;
+for RPM based distributions. see `yum installation instructions
+<https://www.percona.com/doc/percona-server/8.0/installation/yum_repo.html>`_.
+Note that in both cases after installing the current percona-release package,
+you’ll need to enable the testing repository in order to install Percona Server
+for MySQL |release|.  For manual installations, you can download from the
+`testing repository directly through our website
+<https://www.percona.com/downloads/TESTING/Percona-Server-80-rc1/>`_.
+
+New Features
+================================================================================
+
+- :psbug:`4550`: Native Partitioning support for MyRocks storage engine
+- :psbug:`3911`: Native Partitioning support for TokuDB storage engine
+- :psbug:`4946`: Add an option to prevent implicit creation of column family in
+  MyRocks
+- :psbug:`4839`: Better default configuration for MyRocks and TokuDB
+- InnoDB changed page tracking has been rewritten to account for redo logging
+  changes in MySQL 8.0.11.  This fixes fast incremental backups for PS 8.0
+- :psbug:`4434`: TokuDB ROW_FORMAT clause has been removed, compression may be
+  set by using the session variable ``tokudb_row_format`` instead.
+
+Improvements
+================================================================================
+
+- Several packaging changes to bring Percona packages more in line with
+  upstream, including split repositories. As you\’ll note from our instructions
```

Comment:
What tool, what instructions?

Reply:
This part has been added by Tyler. I believe he implies the percona-release. The instructions are probably what appears in the Installation section above